src/runner.ts

```diff
@@ -228,15 +209,53 @@ export class TestsRunner {
         cliProcess.on("error", (err) => { reject(err); });
     }
 
-    private getJavaPath(javaRootPath: string): string | undefined {
-        if (!javaRootPath) {
+    private getSOAtestJavaPath(installDir: string): string | undefined {
+        if (!installDir) {
+            try {
+                const soatestcliPath = which.sync('soatestcli');
+                installDir = soatestcliPath.substring(0, soatestcliPath.lastIndexOf('soatestcli') - 1);
+            } catch (error) {
+                installDir = 'can not find soatestcli'; // it will return false when check installDir exist
```

Comment:
Must catch because an exception is thrown when the command line cannot be found.